shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/ColumnExtractor.java

```diff
@@ -54,4 +59,35 @@
         }
         return Optional.empty();
     }
+    
+    /**
+     * Get left and right value if either value of expression is column segment.
+     *
+     * @param expression expression segment
+     * @return column segment collection
+     */
+    public static Collection<ColumnSegment> extractAll(final ExpressionSegment expression) {
+        if (expression instanceof BinaryOperationExpression) {
+            BinaryOperationExpression boExpression = (BinaryOperationExpression) expression;
+            Collection<ColumnSegment> columns = new ArrayList<>();
+            if (Objects.nonNull(boExpression.getLeft()) && boExpression.getLeft() instanceof ColumnSegment) {
```

Comment:
@cheese8 `Objects.nonNull(boExpression.getLeft())` is useless, can we remove it?